javascript/javascript1/week2/homework4.js

```diff
@@ -0,0 +1,32 @@
+//Student manager
+
+const class07Students = [];
+function addStudentToClass(studentName) {
+  // You write code here
+  if (studentName === "") {
+    console.log("insert a valid name");
+  } else if (class07Students.length > 5 && studentName != "Queen") {
+    console.log("Cannot add more students to class 07");
+  } else if (class07Students.includes(studentName)) {
+    console.log(`Student ${studentName} is already in the class`);
+  } else {
+    class07Students.push(studentName);
+  }
+}
+
+function getNumberOfStudents() {
+  // You write code here
+  return class07Students.length;
+}
+
+addStudentToClass("indira");
+addStudentToClass("");
+addStudentToClass("indira");
+addStudentToClass("loco");
+addStudentToClass("lucas");
+addStudentToClass("theodor");
+addStudentToClass("pål");
+addStudentToClass("pele");
+addStudentToClass("Queen");
+console.log(class07Students);
+getNumberOfStudents();
```

Comment:
This final part of the assignment asks:
`Call the getNumberOfStudents to log out the number of students.`

As it is now, you are printing out the `class07Students` array.

`console.log(getNumberOfStudents());` instead